socialhome/views.py

```diff
@@ -15,6 +17,11 @@ class HomeView(TemplateView):
 
     def get(self, request, *args, **kwargs):
         """Redirect to user detail view if root page is a profile or if the user is logged in"""
+
+        if settings.SOCIALHOME_HOME_VIEW:
```

Comment:
Just want to check that you feel it's better that we don't respect the user setting for landing page if the custom home view is set? Is this what was intended?

Also, it might be better to not render the setting in the case that custom home view is set, otherwise I think it would confuse users since their setting would have no effect at all. Unfortunately, that is tricky since we don't have a proper custom form for that, it comes from `django-dynamic-preferences` app. Would probably require overriding the form and then excluding the field there in init or something.

Something to think about. I'm fine with leaving this detail to later, when users complain :) But maybe think whether you intended to override the user landing page totally or just redirect for non-authenticated users to the custom home view.

Reply:
Our module was convinced as corruption monitor, user opens landing page and see that this city became red (corruption here increasing) , and that district became green (good news). You can play with it on our test server http://test.acts.pp.ua:8005/  (sometimes it unavailable)

So this custom view will be in use by both registered and unregistered users (I think any module of such kind will need focus on custom view). In current sh implementation, respecting user setting will make custom view unavailable for registered users at all, except by logging off.

The best way is to find some solution, that dynamic-preferences will support custom view (and pod admin will have ability to change default view for user). As I see, it could be a little tricky.
Eventually, project is still in development, maybe in future some cool solution will become obvious, and we can leave it as is)